component/dependencies/cloudscale-metrics-collector/component/main.jsonnet

```diff
@@ -0,0 +1,108 @@
+local kap = import 'lib/kapitan.libjsonnet';
+local inv = kap.inventory();
+local params = inv.parameters.cloudscale_metrics_collector;
+
+local namespace = 'appuio-cloud-reporting';
```

Comment:
Usually the namespace is configurable as well.

Reply:
Fixed.